README.md

```
@@ -62,39 +62,50 @@ pip install rms-metadata-tools
-    geometry_config.py
-    <collection>_index.py
-    <collection>_geometry.py
-    <collection>_cumulative.py
+The supplemental index table is controlled by the supplemental label template. By default,
+each column object in the label specifies the name of a PDS label field to add to the table,
+along with its desired formatting. This behavior may be overridden by adding a key function
+to index_config.py of the form:
 
- 3. Create a templates/ subdirectory and copy the label templates from an existing host,
-    and rename accordingly, yielding:
+    key__<NAME>(label_path, label_dict)
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)